tests/unit/fuzz.c

```diff
@@ -24,8 +24,6 @@ afl-fuzz -i fuzzing-examples -o fuzzing-results -- fuzzing/sentry_fuzz_json @@
 #    define _CRT_SECURE_NO_WARNINGS
```

Comment:
I’m wondering if we can remove this whole block, since that is essentially what you need so a direct include of `sentry.h` does not break the build.

Reply:
I'll give this a spin to see what sort of effect this has on our checks. If we fail on compiler warnings then this might not be an option.

Reply:
So removing this doesn't really break anything as far as the CI is concerned, but after looking into the other directives in this block it looks like some of them help with compilation times, which seems like a pretty nice thing to have. I don't know if these fuzzer tests are being run on Windows not to mention the impact removing these would have on Windows builds, so I'm going to leave this in for the time being just to be safe.